elk/training/train.py

```diff
@@ -11,7 +11,7 @@
 from simple_parsing import subgroups
```

Comment:
The function apply_to_layer was already a bit long. I think we should refactor it a bit in a second pull-request